lib/Firtool/Firtool.cpp

```diff
@@ -304,6 +304,7 @@ LogicalResult firtool::populateLowFIRRTLToHW(mlir::PassManager &pm,
 
 LogicalResult firtool::populateHWToSV(mlir::PassManager &pm,
                                       const FirtoolOptions &opt) {
+  pm.addPass(verif::createStripContractsPass());
```

Comment:
Could you nest to any operation to reduce overhead?

Reply:
Great point! Thanks :-)

Reply:
Done!